projects/packages/my-jetpack/_inc/components/product-card/action-buton.jsx

```diff
@@ -66,13 +77,25 @@ const ActionButton = ( {
 			);
 		case PRODUCT_STATUSES.NEEDS_PURCHASE:
 			return (
-				<Button { ...buttonState } size="small" weight="regular" onClick={ onAdd }>
+				<Button
+					{ ...buttonState }
+					href={ `#/add-${ slug }` }
```

Comment:
Not a blocker for me, but maybe we could pass this URL as a parameter, to get more flexibility. Not a requirenment though, just a feeling.

Reply:
I don't see what you have in mind. Could you add an example?

Reply:
It's actually not so crucial, you can disregard this comment